_studio/hevc_fei/h265_fei/include/mfx_h265_fei_encode_plugin_hw.h

```diff
@@ -158,6 +158,12 @@ namespace MfxHwH265FeiEncode
         ~H265FeiEncodePlugin()
         {}
 
+        static H265FeiEncodePlugin* GetInstance()
+        {
+            static H265FeiEncodePlugin instance(false);
```

Comment:
Probably better to use m_adapter instead of static object creation?

Reply:
This is static function it can be called even without any real H265FeiEncodePlugin instance existing, so you can't use H265FeiEncodePlugin::m_adapter here